js/forum/src/components/EmojiAreaButton.js

```diff
@@ -8,23 +8,18 @@ import icon from "flarum/helpers/icon";
 
 export default class EmojiAreaButton extends Component {
 
-    init() {
-        this.textEditor = null;
```

Comment:
Well say you forgot to pass the prop, your way it wouldn't error until you pressed the button. If you call it in the init(), it will become required and will error upon init()